www/sidebars.js

```diff
@@ -6,46 +6,53 @@ module.exports = {
       collapsed: false,
       items: [
         'main/introduction',
-        'react/nextjs',
```

Comment:
If you remove a route, setup a 301 redirect to the new page

Reply:
Just because the "page reference" in sidebars.js changed doesn't mean the actual URL changes — the slug and the "page ID" are separate properties. In this case, I moved the Markdown file `react/nextjs` to `nextjs/intro` but the url is unchange: `/docs/nextjs`.

There are only two instances where the URL did change and they're both pretty minor. 

- `/docs/react-queries-and-mutations` -> `/docs/react-queries`
- `/docs/create-react-app` -> `/docs/react`

I was having some trouble figuring out the deployment pipeline here. I think it's using GitHub pages? GitHub Pages doesn't support redirects, it appears. Can you add these yourself? Unfortunately I really don't have any more time to devote to this.

Reply:
All good, it's deployed with vercel, will add redirects